node_modules/.yarn-state.yml

```diff
@@ -1,10 +0,0 @@
-# Warning: This file is automatically generated. Removing it is fine, but will
-# cause your node_modules installation to become invalidated.
```

Comment:
Was this deletion intentional?

Reply:
No. Why isn't this gitignored?

Reply:
It's intentional to keep that directory in the root

Reply:
Restored.

Reply:
It doesn't seem to get regenerated, so I guess it doesn't matter?